README.md

```
@@ -1,5 +1,9 @@
 # Semver
 
+<p align="center">
+	<img src="docs/logos/semver-github.png" width=400px>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)